CorsixTH/Lua/languages/japanese.lua

```diff
@@ -0,0 +1,2440 @@
+      [2] = "あなたの緊急事態対応は卓越しています。 この最優秀緊急対応賞は、多くの病気や苦しみに対処する最高の能力を示したことを称えています。",
+    },
+    penalty = {
+      [1] = "あなたの緊急時の対応はお粗末でした。 急患は正確で適切な処置を必要としてますが、あなたはそれを提供できませんでした。",
+    },
+  },
+  happy_patients = {
+    awards = {
+      [1] = "昨年、あなたの病院に訪れた人たちはとても満足していました！ 誇らしいですね。",
+      [2] = "ゲーム内の他のどの病院よりも、患者達はあなたの病院の治療にとても満足していました！",
+    },
+    penalty = {
+      [1] = "治療を受けた人は、この病院のひどさに皆、顔をしかめています。 保健省の尊敬を得たいのであれば、もっと良い結果を出す必要があります。",
+      [2] = "あなたの病院の患者たちは皆、この病院を最低だと言います。 患者の幸せをもっと考慮するべきです。",
+    },
+  },
+  high_rep = {
+    awards = {
+      [1] = "上出来です。 昨年、かなり印象的な評判を築いた功績により小さな賞を受賞しました。",
+      [2] = "素晴らしい！ 昨年度最高の評判を得たことにより賞を受賞しました！",
+    },
+    regional = {
+      [1] = "いずれの病院よりも最高の評判を受けた功績で、ブルフロッグ賞をお受け取りください。 最高栄誉賞の受賞おめでとう！",
+      [2] = "あなたの病院は今年度、他病院全てを合わせてもかなわないほどの評判を得ました！ 大きな成果です！",
+    },
+    penalty = {
+      [1] = "昨年を通して非常に悪い評判を維持していたことで、懲戒処分を受けます。 改善を期待します。",
+      [2] = "あなたの病院の評判は地域で最悪です。 恥ずかしいったらありゃしません。 もっと努力するか、転職先を探してください。",
+    },
+  },
+  hosp_value = {
+    awards = {
+      [1] = "保健省は、あなたの病院の高い病院価値を表彰しています！ おめでとうございます！",
+    },
+    regional = {
+      [1] = "あなたは財テク王です！！ あなたの病院は、地域内の他のすべての病院を合わせたよりも価値があります。",
+    },
+    penalty = {
+      [1] = "あなたの病院の価値はお世辞にも高いとは言えません。 財政面での失敗が原因です。 良い病院は高価な病院であることを忘れないでください。",
+    },
+  },
+  wait_times = {
+    award = {
+      [1] = "おめでとうございます。 あなたの病院は一貫して待ち時間が非常に短いと評価されています。 これは重要な賞です！",
+    },
+    penalty = {
+      [1] = "待ち時間があなたの病院では長すぎます。常に許容できない行列があります。もっと患者の整理をうまくやってください。",
+    },
+  },
+  cleanliness = {
+    regional_good = {
+      [1] = "あなたの病院は地域で最も清潔だと評価されています。 作業員たちは昇進に値します。 あなたも尊敬されて然るべきです。",
+    },
+    award = {
+      [1] = "検査官はあなたの病院が非常に清潔であると評価しています。 清潔な病院は安全な病院です。 引き続き作業員たちにこの良い仕事を続けさせてください。",
+    },
+    regional_bad = {
+      [1] = "あなたの病院は地域で１番の不潔さを誇っています。 周辺のすべての病院は廊下をもっと清潔に保っています。 あなたは医療界にとって恥です。 なんとかしてください。",
+    },
+  },
+  healthy_plants = {
+    awards = {
+      [1] = "昨年、あなたは観葉植物を見事に育てたことを評しまして、緑育肥料賞を授与いたします。",
+    },
+    trophies = {
+      [1] = "過去12ヶ月間にわたりすべての植物を良好な状態に保った功績で、観葉植物の友の会より、グリーントロフィー・オブ・ヘルスを授与いたします。",
+      [2] = "昨年、病院の植物を最良の健康状態に保った功績で、グリーンフィンガートロフィーをお受け取りください。",
+    },
+  },
+  happy_vips = {
+    trophies = {
+      [1] = "昨年病院を訪れたVIP達に好印象を与えたことにより、ノーベル賞を送ります。 皆あなたの病院に非常に好意的なコメントを寄せました。",
+      [2] = "有名人エージェンシーから、あなたの病院を訪れたすべてのVIPを満足させたことで、セレブリティトロフィーを授与されることをお祝いします。 少し有名になり、ほぼ私たちの仲間入りです。",
+      [3] = "あなたの病院を訪れたすべての著名人の生活を向上させたことで、VIP訪問賞を受賞しました。 素晴らしい！",
+    },
+  },
+}
+
+
+bank_manager = {
+  current_loan = "Current Loan",
+  interest_rate = "Interest Rate",
+  interest_payment = "Interest Payment",
+  hospital_value = "Hospital Value",
+  statistics_page = {
+    date = "Date",
+    money_in = "In",
+    money_out = "Out",
+    balance = "Balance",
+    current_balance = "Current Balance",
+    details = "Details",
+  },
+  balance = "Balance",
+  insurance_owed = "Insurance Owed",
+  inflation_rate = "Inflation Rate",
```

Comment:
These aren't translated, not sure if @ShimaKen has translations for these; otherwise this section can be squashed out.

Reply:
That's true. I have added the ones and made tried other lines better.
But I can't "commit" the modified file.
I am very new for GitHub... What do i need to do/have?
My updated lua file is here.
[japanese.zip](https://github.com/user-attachments/files/17735028/japanese.zip)

Reply:
@MandrakeHorse are you able to push in the changes? You'll need to check for whitespace again

Reply:
Leave it to me, I can do it